wrongsecrets-balancer/package.json

```diff
@@ -11,6 +11,9 @@
   "author": "Project OWASP WrongSecrets",
   "license": "Apache-2.0",
   "dependencies": {
+    "@google-cloud/iam": "^1.0.1",
+    "@google-cloud/secret-manager": "^5.0.1",
+    "@google-cloud/storage": "^7.1.0",
```

Comment:
can this give problems for aws/azure deployments? (e.g. do we have to update teh build to create flavors of the balancer?)

Reply:
No, I don't think they will cause any harm. It may be more efficient to create different flavors as those packages will not be needed if not running in gcp.